pages/sdks-tools.mdx

```diff
@@ -1,4 +1,208 @@
 ---
```

Comment:
Should we use the description for SDKs and Tools as listed here? 
https://docs.google.com/document/d/1GtCZINh-W5YqVknbibBEm12su94tVDQKp2RMXfyAgug/edit#

Reply:
Didn't see that. Updated!